tests/unit/endpoint.spec.ts

```diff
@@ -513,6 +513,40 @@ describe("Endpoint", () => {
       );
     });
 
+    test("silence zod parse error option", async () => {
+      const endpoint = defaultEndpointsFactory.build({
+        method: "post",
+        input: z.object({
+          limit: z.preprocess(
+            (a) => parseInt(z.string().parse(a), 10),
+            z.number().positive().max(100)
+          ),
```

Comment:
Instead all that, you just need more loose schema in your project, like this one:

```ts
z.number().or(
  z.string().transform(
    (str) => parseInt(str, 10)
  )
)
```

Reply:
this way, you accept both `number` and `string`, but in any case you supply `number` to your handler.